spec/models/app_config_spec.rb

```diff
@@ -44,4 +51,24 @@
     end
     @ac.favicon_content_type = 'image/vnd.microsoft.icon'
   end
+
+  context '.contact_email' do
+    it 'returns the contact e-mail if it is set' do
+      @ac.contact_link_location = 'contact@example.com'
+      @ac.save
+
+      expect(AppConfig.contact_email).to eq('contact@example.com')
+
+      AppConfig.delete_all
```

Comment:
`before(:all)` runs *once* before all of the tests in the block, if we were running it `before(:each)` that would be the same but this follows the [Four Phase Test](https://robots.thoughtbot.com/four-phase-test) model (with `AppConfig.delete_all` being "teardown").